addons/mail/models/ir_actions.py

```diff
@@ -51,6 +66,23 @@ def run_action_email(self, action, eval_context=None):
         action.template_id.send_mail(self._context.get('active_id'), force_send=False, raise_exception=False)
         return False
 
+    @api.model
+    def run_action_next_activity(self, action, eval_context=None):
+        if not action.activity_type_id or not self._context.get('active_id'):
+            return False
+
+        record = self.env[action.model_id.model].browse(self._context.get('active_id'))
```

Comment:
Why don't you support multi mode (with active_ids I think), like we do with code-based server actions ?

Reply:
Yes it does make sense. Will change.